### ISSUE TITLE
Displaying sorted users in dropdowns?

### ISSUE THREAD

**Issue by MaximilianKrambach** [OpenSlides/OpenSlides#4363]
Currently, a sorting of users does not affect their display. A sorted list (in dropdowns) is sorted by the config value `users_sort_by`. In 2.3, this affected the display, too. 

A dropdown list sorted by last name is currently:
```
Ximena Alpha
Peter Beta
Dr. John Gamma
```
In 2.3, this was:
```
Alpha, Ximena
Beta, Peter
Gamma, Dr. John
```
Which do we want?

**Comment by tsiegleauq** [comment]
relevant:
https://github.com/OpenSlides/OpenSlides/pull/4390

**Comment by MaximilianKrambach** [comment]
new PR: #4411

**Comment by tsiegleauq** [comment]
#4411 was merged, sorting (usually) happens at a central place in the repository now.
As far as I see, that issue was resolved. Please reopen if I missed something.

**Comment by MaximilianKrambach** [comment]
It is more about 'How do we want it displayed'. This is not about the sorting itself (which was solved)

**Comment by tsiegleauq** [comment]
@emanuelschuetze 
TL;DR Should users in lists be displayed by
`{TITLE} {FIRST_NAME} {LAST_NAME}`
(OpenSlides 3.0)

or
`{LAST_NAME}, {FIRST_NAME}`
(OpenSlides 2.3)

**Comment by emanuelschuetze** [comment]
Display of users works in 3.0 -> won't add a 2.3 display feature in kind of "lastname, firstname".
